ibis/omniscidb/client.py

```diff
@@ -905,6 +905,43 @@ def drop_table(self, table_name, database=None, force=False):
         self._execute(statement, False)
         self.set_database(_database)
 
+    def add_column(self, table_name, **columns_with_types):
+        """
+        Add a given column(s).
+
+        Parameters
+        ----------
+        table_name : string
+        columns_with_types : keyworded arguments (column(s) with type(s))
+
+        Examples
+        --------
+        >>> table_name = 'my_table'
+        >>> columns_with_types = my_column_1 = 'INTEGER',
```

Comment:
it seems this line is raising an error on doctest tests.
maybe you could use a dictionary here.

Reply:
problem was relating to writing multiple lines.